lib/screens_web/controllers/v2/screen_api_controller.ex

```diff
@@ -8,7 +8,7 @@ defmodule ScreensWeb.V2.ScreenApiController do
   alias Screens.V2.{ScreenAudioData, ScreenData}
   alias ScreensConfig.Screen
 
-  @base_response %{data: nil, disabled: false, force_reload: false}
+  @base_response %{data: nil, disabled: false, force_reload: false, flex_zone: nil}
```

Comment:
I'd be fine with omitting this from the base response, and having it only present at all for "Mercury requests", to reduce confusion about what its role is in our "normal" screen pipeline. I think our other Mercury special-case fields basically work the same way.